packages/react-components/config/api-extractor.local.json

```diff
@@ -1,33 +1,5 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
   "extends": "./api-extractor.json",
-  "mainEntryPointFilePath": "<projectFolder>/dist/packages/<unscopedPackageName>/src/index.d.ts",
```

Comment:
this was removed upon running migration script

Reply:
I think the one in react-components needs to be different (so the migration script is wrong in this case)--you can look at the issue that was linked in the comment to see if it's still relevant.

Reply:
heh, this comment in the discussion hid all the lines below being deleted. Thought Geoff was only talking about the 1 line.

Reply:
As for this issue, it might be good for @Hotell to confirm, but I have no problem building to react-components and creating an updated api file without this workaround. So it __seems__ to be resolved?

Reply:
passing!

Reply:
returned this code. I'll make a mention in the linked issue that this appears to be resolved and this should could be removed.

Reply:
yeah react-components is the only exempt as it needs various workarounds because limitations of our current tooling. this will be resolved in future.